README.md

```diff
@@ -213,8 +213,8 @@ const columns = [
 |**`empty`**|boolean|false|This denotes whether the column has data or not (for use with intentionally empty columns)
 |**`viewColumns`**|boolean|true|Allow user to toggle column visibility through 'View Column' list
 |**`filterList`**|array||Filter value list [Example](https://github.com/gregnb/mui-datatables/blob/master/examples/column-filters/index.js)
-|**`filterOptions`**|{names, logic}||Filter options. it's possible use an array of names [Example](https://github.com/gregnb/mui-datatables/blob/master/examples/column-filters/index.js) or an object with names and logic [Example](https://github.com/gregnb/mui-datatables/blob/master/examples/customize-filter/index.js)
-|**`customFilterListRender`**|function||Function that returns a string used as the chip label. `function(value) => string` [Example](https://github.com/gregnb/mui-datatables/blob/master/examples/column-filters/index.js)
+|**`filterOptions`**|{names, logic}||Filter options. An object with names and logic [Example](https://github.com/gregnb/mui-datatables/blob/master/examples/customize-filter/index.js)
+|**`customFilterListRender`**|function||Function that returns a string used as the chip label. `function(value) => string` [Example](https://github.com/gregnb/mui-datatables/blob/master/examples/column-filters/index.js) or it's still possible to use an array of names, this option has been deprecated [Example](https://github.com/gregnb/mui-datatables/blob/master/examples/column-filters/index.js)
```

Comment:
I think the change here was misapplied to `customFilterListRender` instead of `filterOptions`.